.github/workflows/benchmark_on_push.yml

```diff
@@ -18,16 +18,20 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: 3.8
+
       - name: Install Linux system dependencies
         run: |
           sudo apt-get update
           sudo apt install gfortran gcc libopenblas-dev
+
       - name: Install python dependencies
-      # Pin asv==0.5.1 to fix failing benchmarks. Related to https://github.com/airspeed-velocity/asv/issues/1323
         run: |
-          python -m pip install --upgrade pip wheel setuptools virtualenv asv==0.5.1 wget cmake casadi numpy
-      - name: Install SuiteSparse and Sundials
+          python -m pip install --upgrade pip wheel setuptools virtualenv wget cmake casadi numpy virtualenv packaging
+          python -m pip install asv[virtualenv]
```

Comment:
We can get rid of `virtualenv` from one of these lines.

Reply:
Done